packages/app/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": ["../../tsconfig.json"],
+  "extends": "../../tsconfig.json",
```

Comment:
Apparently this should be correct, but maybe I looked at wrong version docs? 
I got some weird errors trying to run ts-node to compile a small utility, which lead to this being incorrect actually...in that environment at least